src/skmultiflow/core/stream.py

```diff
@@ -0,0 +1,28 @@
+from streamz import Stream
+
+@Stream.register_api()
+class predict(Stream):
```

Comment:
This is a method reserved for learning methods.

Reply:
I am not convinced about a "more explicit name", if there is a problem then it can certainly be fixed via a different way to call the method ... 

I was thinking of something like this, which would minimize change on the existing code

```python
import functools
def stream_predict(est, stream):
     # streaming dataframe instead of stream ?
     pred_stream = Stream().map(est.predict)
     pred_stream.connect(stream)
     return pred_stream

def make_stream_estimator(est):
    predict = functools.partial(stream_predict, est=est)
    setattr(est, 'stream_predict', predict)
    return est
    # similar for partial_fit
...
...
from sklearn.linear_model import LogisticRegression  
lr = make_stream_estimator(LogisticRegression())
lr.stream_predict(some_stream).map(print)  # print out predictions
```